k8s/deployment/kubernetes.yaml

```diff
@@ -188,7 +258,7 @@ spec:
     spec:
       containers:
         - command:
-            - "chroma"
+            - "coordinator"
             - "coordinator"
```

Comment:
Nit: I feel a little weird about these commands being `coordinator coordinator` and `logservice logservice`. Could we make them `coordinator run` and `logservice run` or something like that?